bolt-client/Cargo.toml

```diff
@@ -1,21 +1,24 @@
 [package]
-name = "bolt-spammer-helder"
+name = "bolt-rpc-client"
 version = "0.1.0"
```

Comment:
This actually counts for the sidecar as well, but on the next release we should bump these versions!